src/MobileDevice/widget/MobileDevice.webmodeler.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, DOM } from "react";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<{}, {}> {
+
+    render() {
+        return DOM.div({});
```

Comment:
All webmodeler widget that are rendered as empty hidden/ should at least set a class so everyone one know it was on purpose. 
Add `className: "mobile-device-no-preview"`